roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -283,6 +289,17 @@ def get_var(self, *keys, **kwargs):
                 ))
 
     @staticmethod
+    def normalize(name_list):
```

Comment:
I guess we could. It feels a little weird but I guess if the action plugin is importing the class anyway it could also use a static method from it.

Don't really want to redo the PR for that though :) I can just add it to my long list of little tech debt things to do.